### DISCUSSION TITLE
Stuck on Splash Screen

### DISCUSSION THREAD

**Discussion by Shaddynotshady** [Imperial-lord/mapbox-flutter#14] [General]
i am using your code mapbox complete turn by turn but its stuck on splash screen 
![WhatsApp Image 2022-05-31 at 11 47 44 AM (1)](https://user-images.githubusercontent.com/95290684/171263351-6cb6c9ab-6467-4211-beb6-ab9d4e6be10a.jpeg)
![Screenshot (64)](https://user-images.githubusercontent.com/95290684/171263402-89f1b994-bf27-4199-9b19-a2e6da7325c4.png)

**Comment by Imperial-lord** [comment]
What is the stack trace? This could be an internet issue too. Let me know.
